conversation_classification/prediction_utils/features2vec.py

```diff
@@ -346,8 +385,31 @@ def _get_term_features(actions, UNIGRAMS_LIST, BIGRAMS_LIST):
 # Extracts comment level conversational features from the last N comments
 # N determined by parameter cnt
 def _get_last_n_action_features(actions, cnt, LEXICONS):
+    """
+      Extracts 1) utterance length features and 2) linguistic features at utterance level for last N utterances.
+          - Parameters: 
+               - cnt: N
+               - LEXICONS: dictionary of predefined lexicons
+          - Features colloected:
```

Comment:
Typo: should be "collected"

Reply:
Also should specify that this returns a dictionary containing the following features...